### ISSUE TITLE
Fix prefLabel of IntergovernmentalOrganization

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#756]
```
gist:IntergovernmentalOrganization
	a owl:Class ;
	skos:prefLabel "Inter-Governmental Organization"^^xsd:string ;
	.
```

"Intergovernmental" is one word - as made clear in the local name. PrefLabel should be "IntergovernmentalOrganization."

**Comment by rjyounes** [comment]
Fix with #766.

**Comment by rjyounes** [comment]
@DanCarey404  Please roll this into the fix for #766.
